pkg/analyze/host_system_packages.go

```diff
@@ -120,13 +120,23 @@ func (a *AnalyzeHostSystemPackages) Analyze(getCollectedFileContents func(string
 }
 
 func getSystemPackageTemplateMap(pkg collect.SystemPackage, osName string, osVersion string) map[string]interface{} {
+	osVersionParts := strings.Split(osVersion, ".")
```

Comment:
`strings.SplitN(osVersion, ".", 3)`

Reply:
from the docs:
```
n > 0: at most n substrings; the last substring will be the unsplit remainder.
```
so how would that help?